fixitfriday.ui/.editorconfig

```
@@ -0,0 +1,11 @@
+# stop .editorconfig files search on current file.
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
```

Four is the Ed-Fi standard (see [item #1 under Coding Style](https://techdocs.ed-fi.org/display/EDFIODS/C%23+Coding+Standards+-+Draft+Revision)).
```suggestion
indent_size = 4
```

---

Well, based on what we talked, I think we'll leave it on two for now